src/PowerSystems.jl

```diff
@@ -21,6 +21,7 @@ export AreaInterchange
 export get_aggregation_topology_accessor
 export SupplementalAttribute
 export GeographicInfo
+export get_geo_json
```

Comment:
I do not see this function in the branch psy5. Am I missing something?

Reply:
its imported from IS